### ISSUE TITLE
Not able to update num_top_classes for Label component

### ISSUE THREAD

**Issue by venkyyuvy** [gradio-app/gradio#5068]
### Describe the bug

Is there any reason for not allowing the `num_top_classes` to be updated using the `update` method for `Label` component?

I would like to dynamically choose this value based on user input. Kindly advise, if there are any other ways to achieve this. Thanks.

### Have you searched existing issues?  🔎

- [X] I have searched and found no existing issues

### Reproduction

gr.Label.update(top_num_classes=x)

### Logs

```shell
TypeError: update() got an unexpected keyword argument 'top_num_classes'
```

### System Info

```shell
Mac
```

### Severity

Blocking usage of gradio

**Comment by freddyaboulton** [comment]
This will be fixed by #5066

**Comment by abidlabs** [comment]
Hi @venkyyuvy this is now fixed. On `main`, this will work for you:

```py
import gradio as gr

with gr.Blocks() as demo:
    l = gr.Label({"a": 0.7, "b": 0.2, "c": 0.1})
    btn2 = gr.Button("Show top 2")
    btn3 = gr.Button("Show top 3")
    btn2.click(lambda : gr.Label({"a": 0.7, "b": 0.2, "c": 0.1}, num_top_classes=2), None, l)
    btn3.click(lambda : gr.Label({"a": 0.7, "b": 0.2, "c": 0.1}, num_top_classes=3), None, l)

demo.launch()
```

Note that right now, the value of label needs to be passed again -- but we are working on fixing that.

(You can install gradio from main by following the instructions here: https://gradio.app/docs/main)
